docs/repositories.md

```diff
@@ -32,6 +32,7 @@ This is necessary because you can:
 To clone your new repository to your machine, [follow these instructions](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) which explains cloning both via GitHub Desktop or via the command line.
 When this is done, you should have a folder whose name is the same as the repo on your machine.
 
+Note that if someone else wants to commit to your recently created OpenSAFELY repo, they may need to wait up to 30 minutes for the necessary write permissions to be granted.
```

Comment:
The script runs hourly, meaning an average of ~30min, but could be up to an hour

Reply:
I've amended to say "up to an hour".